resources/definitions/ultimaker_sketch.def.json

```diff
@@ -0,0 +1,187 @@
+        "fill_outline_gaps": { "value": false },
+        "gantry_height": { "value": "60" },
+        "infill_angles": { "value": "[45,45,45,45,45,135,135,135,135,135]" },
+        "infill_before_walls": { "value": false },
+        "infill_overlap": { "value": 0 },
+        "infill_pattern": { "value": "'zigzag'" },
+        "infill_sparse_density": { "value": 20 },
+        "infill_wipe_dist": { "value": 0 },
+        "initial_layer_line_width_factor": { "value": 125 },
+        "inset_direction": { "value": "'inside_out'" },
+        "jerk_enabled":
+        {
+            "enabled": false,
+            "value": false
+        },
+        "layer_height_0": { "value": "layer_height * 1.25" },
+        "layer_start_x": { "value": "sum(extruderValues('machine_extruder_start_pos_x')) / len(extruderValues('machine_extruder_start_pos_x'))" },
+        "layer_start_y": { "value": "sum(extruderValues('machine_extruder_start_pos_y')) / len(extruderValues('machine_extruder_start_pos_y'))" },
+        "machine_center_is_zero": { "default_value": true },
+        "machine_depth": { "default_value": 150 },
+        "machine_end_gcode": { "default_value": "M107; Disable Fan; \n; End of print; \n; End GCode\nM104 S0 T0; Set Toolhead Temp to 0\nM140 S0 T0; Set Platform Temp to 0\nG162 Z F1800; Move to max axes position\nG28 X Y; Home\nM652; Turn off back fan\nM132 X Y Z A B; Set Home Position\nG91; Use Relative Positioning\nM18; Disable Axes\n\n" },
+        "machine_extruder_count": { "default_value": 1 },
+        "machine_gcode_flavor": { "default_value": "Griffin" },
+        "machine_heated_bed": { "default_value": true },
+        "machine_height": { "default_value": 150 },
+        "machine_max_feedrate_x": { "default_value": 300 },
+        "machine_max_feedrate_y": { "default_value": 300 },
+        "machine_max_feedrate_z": { "default_value": 40 },
+        "machine_min_cool_heat_time_window": { "value": "15" },
+        "machine_name": { "default_value": "MakerBot Sketch" },
+        "machine_nozzle_cool_down_speed": { "default_value": 0.8 },
+        "machine_nozzle_heat_up_speed": { "default_value": 1.4 },
+        "machine_start_gcode": { "default_value": "M140 S50 T0; Set Platform Temp\nM104 S220 T0; Set Extruder Temp\nG90; Use Absolute Positioning\nG28; Home\nM132 X Y Z A B; Set Current Position to Home\nG161 X Y F3300; Move to min axes positions\nM7 T0; Wait For Platform to Heat\nM6 T0; Wait For Extruders to Heat\nM651; Turn on back fan\nM907 X100 Y100 Z40 A80 B20; Set Stepper Currents\nM106; Enable Cooling Fan\n; Purge Line\nG92 E0; Reset Extruder Axis Position\nG1 X-26.18 Y-75.90 Z0.200 F420\nG1 X26.18 Y-75.90 E10\nG92 E0; Reset Extruder Axis Position\n; Start GCode\n" },
+        "machine_width": { "default_value": 150 },
+        "material_diameter": { "default_value": 1.75 },
+        "material_flow": { "default_value": 100 },
+        "min_bead_width":
+        {
+            "default_value": 0.4,
```

Comment:
This default value will be ignored as you also provide a value (fails unit test)
```suggestion
```

Reply:
fixed in latest commit